Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1423,9 +1822,20 @@ instance : Sub PGame :=
   ⟨fun x y => x + -y⟩
 
 @[simp]
-theorem sub_zero (x : PGame) : x - 0 = x + 0 :=
+theorem sub_zero_eq_add_zero (x : PGame) : x - 0 = x + 0 :=
   show x + -0 = x + 0 by rw [neg_zero]
 
+protected lemma sub_zero (x : PGame) : x - 0 ≡ x :=
+  _root_.trans (of_eq x.sub_zero_eq_add_zero) x.add_zero
+
+/-- Use the same name convention as global lemmas. -/
```

Comment:
This doc string doesn't explain the lemma, and will be confusing when it shows up by itself on a hover.